### ISSUE TITLE
Porting woes: INITENV was originally paragraph-aligned already

### ISSUE THREAD

**Issue by ecm-pushbx** [SvarDOS/edrdos#96]
The main change in https://github.com/SvarDOS/edrdos/commit/a9adbbf2fb4077a8eff2e443b428da8c3e803623 just undoes what https://github.com/SvarDOS/edrdos/commit/f4e868e1ade78bb55fd72abac02cede50ad91794#diff-52bbf87db189a4af9f24037eb12b3b9563a54bfc343b81073ad25c86fbf9f5dcL111 did!

**Comment by boeckmann** [comment]
Yes, wondering why it was not implicitly para-aligned by the previous definition in BIOSGRPS. There it is para-aligned. And this should have propagated to the INITENV in the genercfg.asm.

**Comment by boeckmann** [comment]
Btw @ecm-pushbx do you think we can safely remove the 256 byte patch area filled by NOP? To my understanding, we can simply rebuild the kernel if there is something to patch?

**Comment by ecm-pushbx** [comment]
I assume you mean https://github.com/SvarDOS/edrdos/blob/83b66305d6498cad55c585ede6a6a639cbc3fdc1/drbio/biosinit.asm#L292 and https://github.com/SvarDOS/edrdos/blob/83b66305d6498cad55c585ede6a6a639cbc3fdc1/drdos/header.asm#L1392 ?

I don't know what exactly this was intended for. You're of the opinion it is for hot patching existing kernel files, to ease this sort of patch? If that is indeed the only use then yeah I'd say drop it.

**Comment by boeckmann** [comment]
Exactly, found no other reference to it in the source, so I assume this is for hot patching. But I am not 100% sure. Who knows what legacy DOS program could depend on this?

**Comment by ecm-pushbx** [comment]
It is no more than 512 Bytes, maybe less (not sure whether the bio patch is init only area or resident). So I'd rather be careful for now.
